src/Language/Mulang/Operators/Haskell.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE ViewPatterns #-}
+
+module Language.Mulang.Operators.Haskell (haskellTokensTable) where
+import Language.Mulang.Operators (TokensTable, buildTokensTable)
+import Language.Mulang.Ast.Operator (Operator (..))
+
+haskellTokensTable :: TokensTable
+haskellTokensTable =
+  buildTokensTable [
+    (NotEqual, ["/="]),
+    (Negation, ["not"]),
+    (Otherwise, ["otherwise"]),
+    (BackwardComposition, ["."]),
+    (Divide, ["/", "div"])
```

Comment:
won't this cause ambiguous parsing?

Reply:
Yes, you are right. I will remove the `div` from the list